packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -180,21 +185,18 @@ module.exports = (ipc, pluginsFile, projectRoot) => {
   if (!tsRegistered) {
     try {
       const tsPath = resolve.sync('typescript', {
-        basedir: this.projectRoot,
+        basedir: projectRoot,
       })
 
-      debug('typescript path: %s', tsPath)
+      const tsOptions = tsNodeOptions.getTsNodeOptions(tsPath)
 
-      tsnode.register({
-        compiler: tsPath,
-        transpileOnly: true,
-        compilerOptions: {
+      debug('typescript path: %s', tsPath)
+      debug('registering plugins TS with options %o', tsOptions)
 
-          esModuleInterop: true,
-        },
-      })
+      tsnode.register(tsOptions)
     } catch (e) {
-      debug(`typescript doesn't exist. ts-node setup passed.`)
+      debug('typescript doesn\'t exist. ts-node setup failed.')
```

Comment:
```suggestion
      debug(`typescript doesn't exist. ts-node setup failed.`)
```

How about removing `\` with "`"?